app/server/lib/sendAppPage.ts

```diff
@@ -26,7 +26,7 @@ import jsesc from 'jsesc';
 import * as path from 'path';
 import difference = require('lodash/difference');
 
-const translate = (req: express.Request, key: string, args?: any) => req.t(`sendAppPage.${key}`, args);
+const translate = (req: express.Request, key: string, args?: any) => String(req.t(`sendAppPage.${key}`, args));
```

Comment:
```suggestion
const translate = (req: express.Request, key: string, args?: any) => toString(req.t(`sendAppPage.${key}`, args));
```
Use `toString` from lodash. String constructor works differently from what most people think. 
 ```
 String(null) === 'null'
 String(undefined) === 'undefined'
```

Reply:
Good catch!

If I may, I'd rather use the optional chaining: `req.t(`sendAppPage.${key}`, args)?.toString()` which I think would give the same result as for lodash's `toString`, if that's fine for you.